packages/contracts-rfq/contracts/FastBridgeV2.sol

```diff
@@ -298,12 +305,21 @@ contract FastBridgeV2 is Admin, IFastBridgeV2, IFastBridgeV2Errors {
         // update protocol fees if origin fee amount exists
         if (transaction.originFeeAmount > 0) protocolFees[transaction.originToken] += transaction.originFeeAmount;
 
-        // transfer origin collateral less fee to specified address
-        address token = transaction.originToken;
-        uint256 amount = transaction.originAmount;
-        token.universalTransfer(to, amount);
+        // transfer origin collateral to specified address (protocol fee was pre-deducted at deposit)
+        if (transaction.originToken == UniversalTokenLib.ETH_ADDRESS) {
+            Address.sendValue(payable(to), transaction.originAmount);
+        } else {
+            IERC20(transaction.originToken).safeTransfer(to, transaction.originAmount);
+        }
 
-        emit BridgeDepositClaimed(transactionId, bridgeTxDetails[transactionId].proofRelayer, to, token, amount);
+        // solhint-disable-next-line max-line-length
```

Comment:
This solhint directive is not really needed :)

Reply:
ah it was until i linted & it stacked the params for me. tx